simphony/cuds/particles.py

```diff
@@ -448,6 +449,19 @@ def _update_element(self, cur_dict, element, clone):
         else:
             raise ValueError('id: {} does not exist'.format(uid))
 
+# ================================================================
+
+    # properties
+
+# ================================================================
+    def _get_data(self):
+        return copy.deepcopy(self._data)
```

Comment:
Please note that the current code is not wrong and my comment does not have to be resolved in this PR.